### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by TaofeeqahBakare** [MDAnalysis/mdanalysis#4486] [Introductions]
Hi everyone! I am Taofeeqah Bakare, a Data Analyst and Biomedical engineering graduate with a keen interest in molecular dynamic analysis. With my analytical background and passion for effective communication, I am excited to make meaningful contributions to help shape MDAnalysis' outreach strategy and connect with the diverse community!

**Comment by adetutudeborah** [comment]
Hello everyone. My name is Adetutu Oluwasanmi, a microbiology graduate with a background in scientific research and communication. I am excited about the opportunity to contribute significantly to the expansion of MDAnalysis' contributor base and its outreach to the target audience.

**Comment by onanugazainab** [comment]
Hello everyone, my name is onanuga zainab, a mechanical engineer and a user researcher. I am so excited to be giving the opportunity to come up with meaningful strategies to help with the expansion of MDAnalysis contributor base and connect with diverse community.

**Reply by TaofeeqahBakare** [reply]
Welcome Zainab! Looking forward to interacting with you. This is the [link](https://discord.gg/fXTSfDJyxE) to join the discord server to access prior conversations.

**Reply by onanugazainab** [reply]
Thank you.

**Comment by hmacdope** [comment]
Hi everyone! Welcome!

**Reply by adetutudeborah** [reply]
Thank you!

**Reply by onanugazainab** [reply]
Thank you!

**Reply by Idowufaith-data** [reply]
Thank you!

**Comment by Idowufaith-data** [comment]
Hello everyone! My name is Faith, a biochemistry graduate turned social media manager/community manager. I’m really excited to contribute to MDAnalysis’ user and contributor base. And I really look forward to collaborating with you all.

**Reply by onanugazainab** [reply]
You are welcome, looking forward to interact with you, https://discord.gg/fXTSfDJyxE, this is the link to the discord group

**Reply by onanugazainab** [reply]
You can join to see our previous conversations, and if you need any help you can reach out to me.

**Reply by Idowufaith-data** [reply]
Thank you @Yasmeenbola, I look forward to interacting with you too.

**Comment by ummyhajj** [comment]
Hello everyone, my name is Ummi Hajaratu Siba and I am privileged and excited to be on this platform. I am a Computer Science graduate with the ambition to become a Data Scientist. I would like to contribute to the project by developing a dynamic communication strategy for the growing MDAnalysis users and contributors.😇
